lib/entity.js

```diff
@@ -0,0 +1,65 @@
+const Request = require('./helpers/request');
+const methods = require('./helpers/methods');
+
+module.exports = class Entity extends Request {
```

Comment:
@wimleers Thanks. At some point I will probably roll the `entity query` work into this, so we don't have two separate ways to get items.